docs/tools/terraform/reference/cookbook/postgres-clickhouse-integration-recipe.rst

```diff
@@ -0,0 +1,160 @@
+
+This recipe calls for the following:
+
+1. Set up an Aiven for ClickHouse database.
+2. Insert your measurements data into the Aiven for ClickHouse database.
+3. Combine your measurements data in the Aiven for ClickHouse database with the related PostgreSQL dimension tables.
+
+Configure common files
+''''''''''''''''''''''
+
+.. dropdown:: Expand to check out the relevant common files needed for this recipe.
+
+  Navigate to a new folder and add the following files:
+
+  1. ``provider.tf`` file
+
+    .. code-block:: terraform
+
+       terraform {
+         required_providers {
+           aiven = {
+             source  = "aiven/aiven"
+             version = ">= 3.7"
+           }
+         }
+       }
+   
+       provider "aiven" {
+         api_token = var.aiven_api_token
+       }
+
+  .. tip::
+
+    You can set environment variable ``AIVEN_TOKEN`` for the ``api_token`` property so that you don't need to pass the ``-var-file`` flag when executing Terraform commands.
+
+  2. ``variables.tf`` file
+
+  Use it for defining the variables to avoid including sensitive information in source control. The ``variables.tf`` file defines the API token, the project name, and the prefix for the service name.
+
+    .. code:: terraform
+
+       variable "aiven_api_token" {
+         description = "Aiven console API token"
+         type        = string
+       }
+   
+       variable "project_name" {
+         description = "Aiven console project name"
+         type        = string
+       }
+
+  3. ``*.tfvars`` file
+
+  Use it to indicate the actual values of the variables so that they can be passed (with the ``-var-file=`` flag) to Terraform during runtime and excluded later on. Configure the ``var-values.tfvars`` file as follows:
+
+    .. code-block:: terraform
+
+       aiven_api_token     = "<YOUR-AIVEN-AUTHENTICATION-TOKEN-GOES-HERE>"
+       project_name        = "<YOUR-AIVEN-CONSOLE-PROJECT-NAME-GOES-HERE>"
+
+Configure the ``services.tf`` file
+''''''''''''''''''''''''''''''''''
+
+The following Terraform script initializes both Aiven for PostgreSQL and Aiven for ClickHouse services, creates the service integration, the source PostgreSQL table, and the Aiven for ClickHouse database.
+
+.. code-block:: terraform
+
+  // Postgres service based in GCP US East
+  resource "aiven_pg" "postgres" {
+    project                 = var.project_name
+    service_name            = "postgres-gcp-us"
+    cloud_name              = "google-us-east-4"
+    plan                    = "business-8" // Primary + read replica
+    maintenance_window_dow  = "monday"
+    maintenance_window_time = "10:00:00"
+  }
+
+  // Postgres sensors database
+  resource "aiven_pg_database" "mydatabase" {
+    project       = var.project_name
+    service_name  = aiven_pg.postgres.service_name
+    database_name = "sensors"
```

Comment:
Change the database name to something more specific, precising that it's comprised of dimension tables
```suggestion
    database_name = "sensor_dims"
```